player/lua/osc.lua

```diff
@@ -672,11 +672,11 @@ function render_elements(master_ass)
 
         elem_ass:merge(style_ass)
 
-        if not (element.type == "button") then
+        if not element.type == "button" then
```

Comment:
Should this be `element.type ~= "button"` ?

Reply:
Yup.